<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createSqliteDatabaseDialog</class>
 <widget class="QDialog" name="createSqliteDatabaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Title:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edTitle">
     <property name="toolTip">
      <string>Descriptive title for the database. 
</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Db Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="txtDbName">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;file name for the database file. If the file extension is not &amp;quot;.sqlite3&amp;quot; it will be appended to what is entered. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPath">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The default location need not be changed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowse">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Browse for database path</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../Resources/mapper.qrc">
         <normaloff>:/fileload.png</normaloff>:/fileload.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="chkCreateConnection">
     <property name="text">
      <string>Create Connection</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>City:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cbCities">
     <property name="toolTip">
      <string>City name. Select an existing name or enter the name of a new city. </string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Company</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edCompany">
       <property name="toolTip">
        <string>Initial Company to be established when a new city is being created. </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dtCompanyStart">
       <property name="displayFormat">
        <string>MM/dd/yyyy</string>
       </property>
       <property name="date">
        <date>
         <year>2018</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dtCompanyEnd">
       <property name="currentSection">
        <enum>QDateTimeEdit::MonthSection</enum>
       </property>
       <property name="displayFormat">
        <string>MM/dd/yyyy</string>
       </property>
       <property name="date">
        <date>
         <year>2050</year>
         <month>3</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>LatLng:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtLatLng">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter latitude and longitude separated by a comma. Or right click on Google Maps at the desired location and paste the value here. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblHelp">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>159</red>
           <green>158</green>
           <blue>158</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources/mapper.qrc"/>
 </resources>
 <connections/>
</ui>
